tutorials/ai-core-setup/ai-core-setup.md

```diff
@@ -332,7 +332,7 @@ List all the resource groups. Check that there is a resource group named **defau
 
 <!-- border -->![image](img/postman/rg.png)
 
-### Troubleshooting
+# Troubleshooting
```

Comment:
This header type with only one # is for the tutorial's title. The existing header type with three # is correct.